.mvn/maven-build-cache-config.xml

```diff
@@ -0,0 +1,83 @@
+<cache xmlns="http://maven.apache.org/BUILD-CACHE-CONFIG/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="http://maven.apache.org/BUILD-CACHE-CONFIG/1.0.0 https://maven.apache.org/xsd/build-cache-config-1.0.0.xsd">
+    <configuration>
+        <enabled>true</enabled>
+        <hashAlgorithm>SHA-256</hashAlgorithm>
+        <validateXml>true</validateXml>
+        <remote enabled="false">
+        </remote>
+        <attachedOutputs>
+            <dirNames>
```

Comment:
I (Informational): this is really the non-standard part that enables the cache to store/restore the `dist` folder.